pom.xml

```diff
@@ -63,6 +60,17 @@
     <!--Maven checks for dependendies from these repos in the order shown in the pom.xml
         This isn't well documented and seems to change between maven versions -MAD 4.9.4 -->
     <repositories>
+        <repository>
+            <id>payara-patched-externals</id>
```

Comment:
Does this further tie us to Payara? Would it be harder to switch to Liberty or Wildfly, for example?

Reply:
Whoops. I asked this twice. Here's the answer from @poikilotherm from https://github.com/IQSS/dataverse/pull/7064#discussion_r452391625

"Yes. But we are pretty bound to Payara anyway, as we rely on their patched lib releases. As long as those don't hit upstream, it would lead to troubles when deploying to Wildfly/Liberty/... anyway."